dllStub/ecmdDllStub.C

```diff
@@ -339,7 +339,9 @@ uint32_t dllQueryArray(const ecmdChipTarget & target, ecmdArrayData & queryData,
   return ECMD_SUCCESS;
 } 
 
-uint32_t dllQueryFileLocation(const ecmdChipTarget & i_target, ecmdFileType_t i_fileType, std::list<std::pair<std::string, std::string> > & o_fileLocations, std::string & io_version) {
+uint32_t dllQueryFileLocation(const ecmdChipTarget & i_target, ecmdFileType_t i_fileType, std::list<ecmdFileLocation> & o_fileLocations, std::string & io_version) {
```

Comment:
It looks like dllNetwork has this function, but it still has the old signature from ecmd14. Should that be updated too?

Reply:
Yea, i noticed but decided to leave well enough alone due to the old signature. I can try to make that change too though.